### ISSUE TITLE
Forbid copying link to an invalid expression

### ISSUE THREAD

**Issue by atifaziz** [Swimburger/NCrontabTester#5]
The “Copy Link” button silently copies a link even if the entered expression is invalid. The “Copy Link” button should be disabled if the expression is invalid.

**Comment by Swimburger** [comment]
This may be a weird case, but wouldn't it be okay to share a broken CRON expression?
If in a project a colleague sends you a CRON expression, but it's not valid, you could send them the link with the invalid expression, and they'll see the error message.
I don't see harm in leaving it this way. What do you think?

**Comment by atifaziz** [comment]
I agree with you, but didn't want to create more work for now. Disabling seemed like the simplest thing to do first so a person doesn't do it _accidentally_. A better option (that I was going to propose as a separate issue) would be to add a checkbox that the user must tick to enable the copying button. That way, it would be a conscious decision (which would be rare) to copy an invalid expression.

**Comment by Swimburger** [comment]
Yes, that would complicate things. The lazy developer in me says to maybe use `window.confirm` to warn them?

**Comment by atifaziz** [comment]
> The lazy developer in me says to maybe use `window.confirm` to warn them?

Definitely, could be a start, but terrible UX. 😉 Is it a lot of work though? We already have a [separate `div` displayed when there is an error](https://github.com/Swimburger/NCrontabTester/pull/6) so why not just add the checkbox in there? May be I am oversimplifying it but can take a crack if you'd like to pass.
